libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -81,8 +82,9 @@ typedef struct RequestItem {
 #define OFFSET(x) offsetof(OVContext, x)
 #define FLAGS AV_OPT_FLAG_FILTERING_PARAM
 static const AVOption dnn_openvino_options[] = {
-    { "device", "device to run model", OFFSET(options.device_type), AV_OPT_TYPE_STRING, { .str = "CPU" }, 0, 0, FLAGS },
-    { "nireq",  "number of request",   OFFSET(options.nireq),       AV_OPT_TYPE_INT,    { .i64 = 0 },     0, INT_MAX, FLAGS },
+    { "device",          "device to run model",           OFFSET(options.device_type),     AV_OPT_TYPE_STRING, { .str = "CPU" }, 0, 0, FLAGS },
+    { "nireq",           "number of request",             OFFSET(options.nireq),           AV_OPT_TYPE_INT,    { .i64 = 0 },     0, INT_MAX, FLAGS },
```

Comment:
just leave device and nireq alone, don't change them.

Reply:
Sorry, will modify